src/docs/patterns/forms.mdx

```diff
@@ -73,14 +73,22 @@ Required / optional fields should be identified according to the Material Design
 
 ## Error Messages
 
-If there is an error with the data entered into a field, the field should indicate to the user that there is a problem. Label and helper text should be changed to a red color and the error message should be displayed in the helper text location below the field (any existing helper text should be replaced with the error message). You may optionally show an additional error icon in the text input.
+If there is an error with the data entered into a field, the field should indicate to the user that there is a problem. Label and helper text should be changed to a red color and the error message should be displayed in the helper text location below the field - any existing helper text should be replaced with the error message. You may optionally show an additional error icon in the text input.
```

Comment:
em dash?

Reply:
My problem is I don't know how to type one without google searching "em dash" and copy pasting :sigh